proposals/0002-ledger-car.md

```diff
@@ -0,0 +1,274 @@
+---
+simd: '0002'
+title: Ledger-CAR file format
+author: Richard Patel (Jump Crypto), Peter Rabbitson (Filecoin), Johnathan Victor (Filecoin)
```

Comment:
Jonathan Victor :)

Reply:
I'd also put Riba and I as Protocol Labs not Filecoin!

Reply:
Embarrassing mistake, sorry about that @jnthnvctr. Fixed now!